### DISCUSSION TITLE
Manga tagger error

### DISCUSSION THREAD

**Discussion by shedman214** [Inpacchi/Manga-Tagger#38] [Q&A]
I get to the last part of the install and I get this error for manga tagger:  
![image](https://user-images.githubusercontent.com/32603273/145876831-846bd439-43cf-4392-87a1-d5309bbb88da.png)

**Comment by ThePromidius** [comment]
That means there is no install.bat in the folder. Reply with the content of C:\Manga Tagger for further assist
